### ISSUE TITLE
[BUG][TGL] Headset audio does not work if SOF and alsa-ucm-conf are applied

### ISSUE THREAD

**Issue by starnight** [thesofproject/linux#2617]
We have laptops ASUS B9400CEA and UX482EG which equip with i5-1135G7 and i7-1165G7.  Tested them with mainline kernel 5.10.0-rc7.  The headset audio works fine with legacy HDA audio driver which includes the [commit eeacd80fcb29 ("ALSA: hda/realtek: Enable headset of ASUS UX482EG & B9400CEA with ALC294")](https://git.kernel.org/pub/scm/linux/kernel/git/torvalds/linux.git/commit/sound/pci/hda/patch_realtek.c?id=eeacd80fcb29b769ea915cd06b7dd35e0bf0bc25).

However, the headset audio is failed with SOF driver: 
* The headphone outputs very very low volume, almost can't hear it.
```
$ aplay -l
**** List of PLAYBACK Hardware Devices ****
card 0: sofhdadsp [sof-hda-dsp], device 0: HDA Analog (*) []
  Subdevices: 0/1
  Subdevice #0: subdevice #0
card 0: sofhdadsp [sof-hda-dsp], device 1: HDA Digital (*) []
  Subdevices: 1/1
  Subdevice #0: subdevice #0
card 0: sofhdadsp [sof-hda-dsp], device 3: HDMI1 (*) []
  Subdevices: 1/1
  Subdevice #0: subdevice #0
card 0: sofhdadsp [sof-hda-dsp], device 4: HDMI2 (*) []
  Subdevices: 1/1
  Subdevice #0: subdevice #0
card 0: sofhdadsp [sof-hda-dsp], device 5: HDMI3 (*) []
  Subdevices: 1/1
  Subdevice #0: subdevice #0

$ aplay -D hw:0,0 ~/Documents/test.wav 
Playing WAVE '/sysroot/home/dev/Documents/test.wav' : Signed 16 bit Little Endian, Rate 44100 Hz, Stereo
Warning: rate is not accurate (requested = 44100Hz, got = 48000Hz)
         please, try the plug plugin
```
* The headset MIC shows up, but no sound is recorded.
```
$ arecord -l
**** List of CAPTURE Hardware Devices ****
card 0: sofhdadsp [sof-hda-dsp], device 0: HDA Analog (*) []
  Subdevices: 1/1
  Subdevice #0: subdevice #0
card 0: sofhdadsp [sof-hda-dsp], device 1: HDA Digital (*) []
  Subdevices: 1/1
  Subdevice #0: subdevice #0
card 0: sofhdadsp [sof-hda-dsp], device 6: DMIC48kHz (*) []
  Subdevices: 1/1
  Subdevice #0: subdevice #0
card 0: sofhdadsp [sof-hda-dsp], device 7: DMIC16kHz (*) []
  Subdevices: 1/1
  Subdevice #0: subdevice #0

$ arecord -D hw:0,0 -f S16_LE -c 2 -r 48000 headsetmic-test2.wav 
Recording WAVE 'headsetmic-test2.wav' : Signed 16 bit Little Endian, Rate 48000 Hz, Stereo
```
Place the dmesg and alsa-info as attachments.
[dmesg.txt](https://github.com/thesofproject/linux/files/5670878/dmesg.txt)
[alsa-info.txt](https://github.com/thesofproject/linux/files/5670882/alsa-info.txt)

**Comment by starnight** [comment]
This still can be reproduced after upgraded alsa-lib, alsa-utils, alsa-ucm-conf and alsa-topology-conf to version **1.2.4** with Linux mainline kernel 5.10.0.

**Comment by starnight** [comment]
Found this can be fixed by upgrading PulseAudio to `v14.0`.
